src/MPGDCylinderBarrelTracker_geo.cpp

```diff
@@ -0,0 +1,563 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Whitney Armstrong
+
+/** \addtogroup Trackers Trackers
+ * \brief Type: **BarrelTrackerWithFrame**.
+ * \author Nivedith Ramasubramanian
+ *
+ * \ingroup trackers
+ *
+ * @{
+ */
+#include "DD4hep/DetFactoryHelper.h"
+#include "DD4hep/Printout.h"
+#include "DD4hep/Shapes.h"
+#include "DDRec/DetectorData.h"
+#include "DDRec/Surface.h"
+#include "XML/Layering.h"
+#include "XML/Utilities.h"
+#include <array>
+#include "DD4hepDetectorHelper.h"
+
+using namespace std;
+using namespace dd4hep;
+using namespace dd4hep::rec;
+using namespace dd4hep::detail;
+
+#include "Math/Vector2D.h"
+using namespace ROOT::Math;
```

Comment:
Can we use only the XYVector (I think that is all that is used)?

Reply:
I meant something like this:
```suggestion
using ROOT::Math::XYVector;
```